docker/docker-py3-kms-minimal/Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV LOGFILE STDOUT
 ENV LOGSIZE ""
 
 COPY ./py-kms /home/py-kms
-
+#hadolint ignore=DL3013,DL3018
```

Comment:
Could you remove that? Currently, we are not using any `Dockerfile` linter... Or do we?

Reply:
I'm using a linter to have better docker images, but no problem to remove that from the PR.